proposals/0357-alpenglow_validator_admission_ticket.md

```diff
@@ -0,0 +1,207 @@
+---
+simd: '0357'
+title: Alpenglow Validator Admission Ticket
+authors:
+  - Wen Xu (Anza)
+  - Roger Wattenhofer (Anza)
+category: Standard
+type: Core
+status: Review
+created: 2025-09-11
+feature: (fill in with feature key and github tracking issues once accepted)
+---
+
+## Summary
+
+This SIMD describes how the validator admission ticket (VAT) collection
+described in SIMD 326 will be implemented. Specifically, how it affects
+validator operation procedures.
+
+Validator admission ticket is a mechanism translating the current cost of
+voting into a similar economic equilibrium for Alpenglow. By charging every
+voting validator 1.6 SOL per epoch, it replaces the current voting fee at ~2
+SOLs per epoch, and reduces the likelyhood there are too many voting validators
+immediately after Alpenglow launches.
+
+Everything specified below are protocol-level changes, they need to be
+implemented by all Solana clients. The client specific data structure
+changes are omitted.
+
+## Dependencies
+
+The Validator Admission Ticket is specified in SIMD 326 (Alpenglow).
+
+The compressed BLS Pubkey in Vote Account is specified in SIMD 185 (Vote
+Account v4)
+
+## Motivation
+
+Adding more voting validators to a blockchain does come with costs. At the
+very least, the votes and corresponding rewards need to be processed by
+every voting validator. Therefore, if an attacker can cheaply start a lot
+of voting validators, it will put more pressure on the chain.
```

Comment:
doesn't necessarily have to be an attacker. Even a well meaning full node exerts a cost on all of the other nodes

Reply:
Done.